core/src/main/resources/jenkins/security/RekeySecretAdminMonitor/message_ru.properties

```
@@ -0,0 +1,25 @@
+pleaseRekeyAsap=\
+  Из-за <a href="https://www.jenkins.io/redirect/rekey" rel="noopener noreferrer" target="_blank">уязвимости</a>, \
+  обнаруженной ранее, необходимо \
+  сменить ключ шифрования, используемый для защиты секретов в файлах конфигурации на диске. \
+  Этот процесс просканирует большую часть директории <code>$JENKINS_HOME</code> (<code>{0}</code>), \
+  найдёт шифрованные данные и сменит ключ. Это займёт некоторое время. \
+  Больше информации о последствиях разных способов выполнения этой операции (или бездействия) \
```

Reading this creates an impression that there are different ways of performing rekey'ing.
My understanding is that there's single way of doing it, but different implications. 
```suggestion
  Больше информации о различных последствиях выполнения этой операции (или бездействия) \
```

---

`for more implications about different ways of doing this (or not doing this.)`
this sounds indeed like different ways of performing (or not) rekeying. Opening the link, then going to "re-keying" link within, seems that the "different ways" mentioned are "right now in the background on a running Jenkins" vs "scheduled on boot".

---

Then `Больше информации о различных способах и последствиях выполнения этой операции...`

---

`Because this involves modifying data in $JENKINS_HOME, the administrator should consider the following trade-offs in choosing when to run this.`
from that same page. I feel like the idea of the sentence is "the implications are somewhat different for the different ways to do it", so I still like my version.